### PR TITLE
ICU-21843 Create README.md for navigating the performance test results

### PR DESCRIPTION
**Pull request by gnrunge**
https://unicode-org.atlassian.net/browse/ICU-21843

See https://github.com/gnrunge/icu-perf/tree/gh-pages for the live page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  Each API invocation iteration iterates 1,114,112 times and calls isAlpha()
+  once at each iteration with the respective code point.
+* Loops: For illustration, let's assume that the calibration phase determines
+  that the above 1,114,112 iterations can be repeated 1,000 times within
+  5 seconds.
+* Passes: Each pass runs 1,600 loops of 1,114,112 iterations and records the
+  execution time of the 1,600 loops. If the number of passes is set to 10,
+  we get 10 performance numbers.
+* Final calculation: The minimum number among the passes is selected and
+  diveded by loops*iterations.
```
Comment (sven-oly) [inline README.md]: diveded --> divided

**Comment by markusicu** [comment]
Please do not use the BRS ticket for this. Use a perf-work ticket.

Please prefix the commit message with the ticket.

**Comment by gnrunge** [comment]
Switched ICU-21900 --> ICU-21843.

File: README.md (hunk 1 above)
Comment (gnrunge) [reply README.md]: Done.

**Comment by markusicu** [comment]
Please prefix the commit message with the ticket.

**Comment by gnrunge** [comment]
Ticket prefixed to commit.

**Review by markusicu — APPROVED** [review]
lgtm

**Comment by markusicu** [comment]
Please also add the standard LICENSE file. Copy it from https://github.com/unicode-org/template-repo and please refer to it from the bottom of the readme like here: https://github.com/unicode-org/template-repo/blob/main/README.md?plain=1

**Pull request merged**